### PR TITLE
Add Footer

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

Adds footer, removes cookies/terms of service and database updated text from right sidebar, other minor styling fixes i found

## Please link the Airtable ticket associated with this PR.

https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recsd2sH7rKT9uxCd

## If applicable, include screenshots of the feature/bugfix introduced by this PR (Please include both desktop and mobile if there is a significant UI change).

Web on explore:
image

Web on data:
image

Mobile on explore:
image

## Describe the steps you took to test the feature/bugfix introduced by this PR.

Tested locally with multiple screen layouts, ensure all hyperlinks worked properly.

## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.

No.

### CODE REVIEW

**Comment by atatmaja** [comment]
Putting as draft bc need french translations, but this is good for a review

File: src/components/shared/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import MultiColorIcon from '../../assets/images/colored-icon.svg'
+import { mobileDeviceOrTabletWidth } from '../../constants'
+import Translate from '../../utils/translate/translateService'
+import { AppContext } from "../../context"
+import { withLocaleUrl } from "../../utils/utils";
+
+export const Footer = () => {
+  const [{updatedAt}] = useContext(AppContext);
+
+  const getTabClass = (tabName: string) => {
```
Comment (brett-stephen) [inline src/components/shared/Footer.tsx:15]: Doesn't look like this func is doing anything, could we remove it?

**Review by brett-stephen — COMMENTED** [review]
Just one comment, but looks good

File: src/components/shared/Footer.tsx (hunk 1 above)
Comment (atatmaja) [reply src/components/shared/Footer.tsx:15]: You right, was an artifact left behind from something i copied in. will remove

**Review by brett-stephen — APPROVED** [review]
looks good

**Pull request merged**